src/content/Angular-CSharp/.template.config/template.json

```diff
@@ -8,8 +8,8 @@
   ],
   "description": "A project template for creating an ASP.NET Core application with Angular",
   "groupIdentity": "Microsoft.DotNet.Web.Spa.ProjectTemplates.Angular",
-  "precedence": "9000",
-  "identity": "Microsoft.DotNet.Web.Spa.ProjectTemplates.Angular.CSharp.7.0",
+  "precedence": "9800",
+  "identity": "Microsoft.DotNet.Web.Spa.ProjectTemplates.Angular.CSharp.8.0",
```

Comment:
We seem to be bumping `precedence` values by `100` in item templates and `1000` in project templates. I'm suggesting here that we slow project templates down to match item templates going forward. Please suggest an alternate bump if you prefer @wtgodbe.

Separately, I don't think this has happened in dotnet/aspnetcore yet. We should do this relatively soon.

Reply:
I'd defer to @MackinnonBuck or @TanayParikh on this one, I think there's documentation on this somewhere that we'd need to update if we changed the pattern

Reply:
@wtgodbe, @MackinnonBuck, and @TanayParikh I'm going to merge this sometime tomorrow morning, likely close to noon, if I don't hear from you first. My intent in changing the item template pattern is mostly to make it more consistent w/ the project template `precedence` values. That is, the new `precedence` is `9800` for both templates in this repo.

Separately, we should bump the `identity` values of item and project templates in dotnet/aspnetcore's main branch to end w/ "8.0" and move all `precedence` values (some of which are way down at `100`) to `9800` to make the values consistent across our templates. That would mean we move to `9900` for our 9.0.0 release but that's far in the future.

That said, the <https://github.com/dotnet/aspnetcore/blob/main/docs/UpdatingMajorVersionAndTFM.md> document covers "updating `precedence` and `identity` elements in all template.json files". It suggests matching the major version times `1000` to determine the new `precedence` values and we're already at higher values 😦

**Maybe we should go to `10000` everywhere now** and make the recommendation be "add `1000` to the previous value, which should be `{major version} x 1000 + 2000`❔

----

According to previous conversations w/ @phenning and @vlada-shubina, the only real requirement is to increase `precedence` values for each new `identity`. Effectively, we're making e.g. `Microsoft.DotNet.Web.Spa.ProjectTemplates.Angular.CSharp.8.0` have precedence over `Microsoft.DotNet.Web.Spa.ProjectTemplates.Angular.CSharp.7.0`.

----

PS. An example in <https://github.com/dotnet/aspnetcore/blob/main/docs/UpdatingMajorVersionAndTFM.md> says "increase it to 80000 for 8.0". I think that's a typo :wink:

Reply:
I'm going to do the aspnetcore portion as part of https://github.com/dotnet/aspnetcore/issues/45492. I'm fine with 9800 as long as Mackinnon/Tanay are - I don't think 10000 is sufficiently better to warrant churn on this PR, it's just a question of whether we do `10000` in net8 or net9, which doesn't seem meaningly different to me.

Reply:
@dougbu I think the 80000 number wasn't a typo - it was my suggestion for having the major version align with the precedence value. For example, a value of 9000 for 7.0 can't be updated to 8000 for 8.0 since that would be decreasing the precedence value, so my suggestion was to add an extra 0 and then keep the number of "padding" zeros constant going forward.

If we're worried about consistency between the templates (the precedence values being different by one or more orders of magnitude) I'm happy with either bumping them all to a value like 80000, or going with your suggestion of `{major version} x 1000 + 2000`.

Reply:
> Edit: I don't really have a particular preference as long as the precedence numbers are consistent and predictable.

Completely agree and I also don't have that much invested in the answer. The choices this time are
- `9800`
- `10000`
- `80000`

Well, I do have a slight preference for `9800` just because that'll mean I don't need to change this PR :wink:

@wtgodbe and @TanayParikh could you please vote❔

Reply:
> The idea was to use 9{major}00 for 8.0 and 9.0, then {major}000 forever after.

This works for me, I agree w/ Mackinnon that the most important thing is to have a consistent rule. Can you update https://github.com/dotnet/aspnetcore/blob/main/docs/UpdatingMajorVersionAndTFM.md with the new scheme?

Reply:
> @wtgodbe and @TanayParikh could you please vote❔

I vote 9800

Reply:
> Can you update https://github.com/dotnet/aspnetcore/blob/main/docs/UpdatingMajorVersionAndTFM.md with the new scheme?

Suggest we do that as part of dotnet/aspnetcore#45492. I'll add a comment there.